src/lib/comms/include/sol-netctl.h

```diff
@@ -268,6 +319,21 @@ enum sol_netctl_service_state sol_netctl_service_get_state(
     const struct sol_netctl_service *service);
 
 /**
+ * @brief Gets the service error
+ *
+ * This function gets the error for the netctl service,
+ * after the service is to give by service monitor.
+ *
+ * @param service The service structure which the name is desired
+ *
+ * @return The error. When error occur during connection or disconnection
```

Comment:
SHould you warn the user that if something goes wrong when trying to get the error (lol), this will return NULL. (Is this possible??)